game.c

```diff
@@ -208,22 +208,28 @@ void init(void) {
 	cam.pitch_deg = 0.0f;
 	cam.yaw_deg = 0.0f;
 	cam.turn_vel = HMM_Vec2(0.0f, 0.0f);
+
+	sapp_lock_mouse(true);
```

Comment:
I think it's probably better if we don't lock the mouse right away, and wait until they click the screen. This is more compatible with web, and won't take the user by surprise on native.